usage-examples/go/atlas-sdk-go/examples/billing/line_items/main.go

```diff
@@ -48,6 +48,10 @@ func main() {
 		errors.ExitWithError(fmt.Sprintf("Failed to retrieve pending invoices for %s", p.OrgId), err)
 	}
 
+	if len(details) == 0 {
```

Comment:
This isn't the correct refactor to avoid exiting with error. The logic that sends the error back to this call site is in `internal/billing/collector.go` - I've left a comment on the relevant lines.